examples/asr/transcribe_speech.py

```diff
@@ -349,6 +354,9 @@ def autocast():
     if type(transcriptions) == tuple and len(transcriptions) == 2:
         transcriptions = transcriptions[0]
 
+    if cfg.return_transcriptions:
+        return transcriptions
```

Comment:
The confidence ensemble itself is run with return_transcription=False, but inner models will have it set to True. You can have a look at the tests to see how it's used here: https://github.com/NVIDIA/NeMo/blob/83af9d84329441c27622477ab946af8c0b9aa8a2/scripts/confidence_ensembles/test_confidence_ensembles.py#L99

Reply:
I would expect this will somehow be integrated with `transcribe_speech.py ` since this is the go-to place for all ASR models offline inference. but this makes sense.